src/leiningen/ring/war.clj

```diff
@@ -169,14 +169,15 @@
                       `(~(generate-resolve destroy-sym)))))))
       :print-meta true)))
 
-(defn create-war [project file-path]
+(defn create-war
+  ^JarOutputStream [project ^String file-path]
```

Comment:
No need to split this onto its own line.

Reply:
ok